applications/mne_scan/plugins/eegosports/FormFiles/eegosportsimpedancewidget.cpp

```diff
@@ -0,0 +1,391 @@
+//=============================================================================================================
+/**
+* @file     eegosportsimpedancewidget.cpp
+* @author   Lorenz Esch <Lorenz.Esch@tu-ilmenau.de>
```

Comment:
Please update the authors,month and year. If you copied this file, add you name to the original authors.